tests/Freckle/App/MemcachedSpec.hs

```diff
@@ -92,9 +93,18 @@ spec = withApp loadApp $ do
 
       msgs <- getLoggedMessagesLenient
       let Just LoggedMessage {..} = NE.last <$> NE.nonEmpty msgs
-      loggedMessageText `shouldBe` "Error deserializing"
-      loggedMessageMeta
-        `shouldBe` KeyMap.fromList
-          [ ("action", String "deserializing")
-          , ("message", String "invalid: \"Broken\"")
-          ]
+      Object loggedMessageMeta ^? key "error" . key "message" . _String
+        `shouldBe` Just "Unable to deserialize: invalid: \"Broken\""
+
+      -- This assertion is far too brittle, but can be useful to un-comment if
+      -- you intend to work on this logic specifically
+      -- Object loggedMessageMeta ^? key "error" . key "stack" . _String . to T.lines
+      --   `shouldBe` Just
+      --     [ "CallStack (from HasCallStack):"
+      --     , "  throwM, called at library/Freckle/App/Memcached.hs:121:30 in freckle-app-1.10.8.0-1ebuZKUCQVI9sAWTLATGfO:Freckle.App.Memcached"
+      --     , "  cachingAs, called at library/Freckle/App/Memcached.hs:92:11 in freckle-app-1.10.8.0-1ebuZKUCQVI9sAWTLATGfO:Freckle.App.Memcached"
+      --     , "  caching, called at tests/Freckle/App/MemcachedSpec.hs:87:15 in main:Freckle.App.MemcachedSpec"
+      --     ]
+      Object loggedMessageMeta
+        ^? key "error" . key "stack" . _String . to (length . T.lines)
+        `shouldSatisfy` maybe False (> 1)
```

Comment:
This would be better, since it'll show the stack itself, and not just the length, on failures

```suggestion
        ^? key "error" . key "stack" . _String . to T.lines
        `shouldSatisfy` maybe False (not . null)
```